bitnami/appsmith/README.md

```diff
@@ -0,0 +1,564 @@
+<!--- app-name: appsmith -->
+
+# appsmith
```

Comment:
It would be more appropriate to change the title to **Appsmith packaged by Bitnami** following the same structure as the other charts. For example: https://github.com/bitnami/charts/tree/main/bitnami/drupal#drupal-packaged-by-bitnami.

Reply:
Done